cumulus/parachains/runtimes/testing/penpal/src/xcm_config.rs

```diff
@@ -104,7 +104,7 @@ pub type LocationToAccountId = (
 	// Straight up local `AccountId32` origins just alias directly to `AccountId`.
 	AccountId32Aliases<RelayNetwork, AccountId>,
 	// Foreign locations alias into accounts according to a hash of their standard description.
-	HashedDescription<AccountId, DescribeFamily<DescribeAllTerminal>>,
+	HashedDescription<AccountId, (DescribeTerminus, DescribeFamily<DescribeAllTerminal>)>,
```

Comment:
`DescribeTerminus` is already included in `DescribeAllTerminal` thus in `DescribeFamily<DescribeAllTerminal>`

it should work with no changes to the existing config:

```suggestion
	HashedDescription<AccountId, DescribeFamily<DescribeAllTerminal>>,
```

Reply:
> it should work with no changes to the existing config

Unfortunately, it won’t. it will fall through this branch with error.

https://github.com/paritytech/polkadot-sdk/blob/007565a7225cba59d236ffc2fdaca2438a3665c1/polkadot/xcm/xcm-builder/src/location_conversion.rs#L142

Reply:
right, ok